Makefile

```diff
@@ -9,92 +9,82 @@ ifeq ($(PKG_TAG),)
+	CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 $(MAKE) app-via-docker
 
 app-via-docker-darwin-arm64:
-	CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 $(MAKE) app-via-docker-goos-goarch
+	CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 $(MAKE) app-via-docker
+
+app-via-docker-windows-arm64:
+	CGO_ENABLED=0 GOOS=windows GOARCH=arm64 $(MAKE) app-via-docker
 
 app-via-docker-windows-amd64:
-	CGO_ENABLED=0 GOOS=windows GOARCH=amd64 $(MAKE) app-via-docker-windows-goarch
-
-victoriametrics-backend-plugin-build: \
-	victoriametrics-backend-plugin-linux-amd64-prod \
-	victoriametrics-backend-plugin-linux-arm-prod \
-	victoriametrics-backend-plugin-linux-arm64-prod \
-	victoriametrics-backend-plugin-linux-386-prod \
-	victoriametrics-backend-plugin-amd64-prod \
-	victoriametrics-backend-plugin-arm64-prod \
-	victoriametrics-backend-plugin-windows-prod
-
-victoriametrics-frontend-plugin-build: \
-	frontend-build
-
-victoriametrics-datasource-plugin-build: \
-	victoriametrics-frontend-plugin-build \
-	victoriametrics-backend-plugin-build
-
-victoriametrics-datasource-plugin-pack:
-	tar -czf victoriametrics-datasource-$(PKG_TAG).tar.gz victoriametrics-datasource \
-	&& sha256sum victoriametrics-datasource-$(PKG_TAG).tar.gz \
-	> victoriametrics-datasource-$(PKG_TAG)_checksums.txt \
-	&& rm -rf ./victoriametrics-datasource
-
-victoriametrics-datasource-frontend-plugin-pack: \
+	CGO_ENABLED=0 GOOS=windows GOARCH=amd64 $(MAKE) app-via-docker
+
+vm-backend-plugin-build: \
+	app-via-docker-linux-amd64 \
+	app-via-docker-linux-arm \
+	app-via-docker-linux-arm64 \
+	app-via-docker-linux-386 \
+	app-via-docker-darwin-amd64 \
+	app-via-docker-darwin-arm64 \
+	app-via-docker-windows-amd64 \
+	app-via-docker-windows-arm64
+
+vm-frontend-plugin-build: frontend-build
+
+vm-plugin-build: vm-frontend-plugin-build vm-backend-plugin-build
+
+vm-plugin-pack-tar: vm-plugin-build
+	mkdir -p dist && \
+	tar -czf dist/victoriametrics-datasource-$(PKG_TAG).tar.gz victoriametrics-datasource && \
+	sha256sum dist/victoriametrics-datasource-$(PKG_TAG).tar.gz > dist/victoriametrics-datasource-$(PKG_TAG)_checksums.tar.gz.txt
+
+vm-plugin-pack-zip: vm-plugin-build
+	mkdir -p dist && \
+	zip -r dist/victoriametrics-datasource-$(PKG_TAG).zip victoriametrics-datasource && \
+	sha256sum dist/victoriametrics-datasource-$(PKG_TAG).zip > dist/victoriametrics-datasource-$(PKG_TAG)_checksums.zip.txt
+
+vm-plugin-cleanup:
+	rm -rf ./victoriametrics-datasource
+
+vm-frontend-plugin-pack: \
 	frontend-pack
 
-victoriametrics-datasource-frontend-plugin-release: \
-	victoriametrics-frontend-plugin-build \
-	victoriametrics-datasource-frontend-plugin-pack
+vm-frontend-plugin-release: \
+	vm-frontend-plugin-build \
+	vm-frontend-plugin-pack
 
-victoriametrics-datasource-plugin-release: \
-	victoriametrics-frontend-plugin-build \
-	victoriametrics-backend-plugin-build \
-	victoriametrics-datasource-plugin-pack
+vm-plugin-release: \
+	vm-plugin-pack-tar \
```

Comment:
those two steps make a build of the package. As for me, this operation should only pack the plugin

Reply:
it can lead to unpredictable consequences as broken bundle in result